en/includes/guides/authentication/conditional-auth/configure-conditional-auth.md

```diff
@@ -79,6 +79,16 @@ To add a new secret:
 
 6. Click **Finish** to complete the creation.
 
+### Use secret in the script
```

Comment:
In below section ideally we should denote that "This name is what will be used in the script to refer to the secret"
`<tr>
            <td>Secret Name</td>
            <td>A meaningful name for the secret. This name is not changeable.</td>
        </tr>`

Reply:
addressed with https://github.com/wso2/docs-is/pull/4773/commits/cc28685065385ccc3454062eccfb6f0d6d5576f9